zio-cache/shared/src/main/scala/zio/cache/ManagedCache.scala

```diff
@@ -85,21 +85,32 @@ object ManagedCache {
     capacity: Int,
     managedLookup: ManagedLookup[Key, Environment, Error, Value]
   )(timeToLive: Exit[Error, Value] => Duration): URManaged[Environment, ManagedCache[Key, Error, Value]] =
+    makeWith(capacity, managedLookup, Clock.systemUTC())(exit => ZIO.succeed(timeToLive(exit)))
+
+  /**
+   * Constructs a new cache with the specified capacity, time to live, and
+   * lookup function, where the time to live can depend on the `Exit` value
+   * returned by the lookup function.
+   */
+  def makeWith[Key, Environment, Error, Value](
```

Comment:
Not sure if reusing the same name as the function above will create an issue with scalac 🤔

Reply:
Just tried to `publishLocal` and it's an issue